zipline/zipline_deprecation_warning.py

```diff
@@ -13,20 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cpython cimport bool
```

Comment:
we don't need it anymore.  it was there to support downstream subclassing of some of our Cython classes, but we don't need to do that anymore.